alpinelib/aws/sagemaker.py

```diff
@@ -1,9 +1,39 @@
 import json
+import re
+from typing import List
 
 import boto3
+import numpy as np
 
 
-def send_to_ml_preprocess(ml_endpoint: str, article_body: str):
+def __preprocess_text(text: str) -> List[str]:
+    """
+    This function tokenizes the text into words then lowercases and strips punctuation from it
+    :param text: string of text to preprocess
+    :return: the tokenized and formatted string as a list
+    """
+    return_list = []
+
+    for word in text.split(' '):
+        stripped_text = re.sub(r'[^\w\s]', '', word.lower())
```

Comment:
could you add a one liner comment on intention of regex to save time for future us trying to look at this?